packages/client/src/errors.ts

```diff
@@ -1,6 +1,5 @@
 import { ServerErrorResponse } from '@grpc/grpc-js';
 import { RetryState, TemporalFailure } from '@temporalio/common';
-export { WorkflowExecutionAlreadyStartedError } from '@temporalio/common';
```

Comment:
We should still export this from client

Reply:
Is it exported elsewhere?

Reply:
Yes, I removed things caught by the duplicate export lint rule. This one is covered by `client/index.js`:

```ts
export * from '@temporalio/common/lib/errors';
```